include/ttmlir/Dialect/TT/IR/TTOpsTypes.h

```diff
@@ -110,6 +110,37 @@ inline DataType elementTypeToDataType(Type elementType) {
   }
   return dtype;
 }
+
+inline std::string getDataTypeName(DataType dtype) {
+  switch (dtype) {
+  case DataType::Float32:
+    return "Float32";
+  case DataType::Float16:
+    return "Float16";
+  case DataType::BFloat16:
+    return "BFloat16";
+  case DataType::BFP_Float8:
+    return "BFP_Float8";
+  case DataType::BFP_BFloat8:
+    return "BFP_BFloat8";
+  case DataType::BFP_Float4:
+    return "BFP_Float4";
+  case DataType::BFP_BFloat4:
+    return "BFP_BFloat4";
+  case DataType::BFP_Float2:
+    return "BFP_Float2";
+  case DataType::BFP_BFloat2:
+    return "BFP_BFloat2";
+  case DataType::UInt32:
+    return "UInt32";
+  case DataType::UInt16:
+    return "UInt16";
+  case DataType::UInt8:
+    return "UInt8";
+  }
+  llvm_unreachable("unhandled data type");
+}
+
```

Comment:
Tablegen autogens this, `::llvm::StringRef DataTypeEnumToString(DataType);`

Reply:
Oh got it. I'll revert that then.